src/js/components/AlertDialogComponent.jsx

```diff
@@ -0,0 +1,72 @@
+var React = require("react/addons");
+var classNames = require("classnames");
+
+var Util = require("../helpers/Util");
+var ModalComponent = require("../components/ModalComponent");
+
+var AlertModalComponent = React.createClass({
+  displayName: "AlertModalComponent",
+
+  propTypes: {
+    data: React.PropTypes.shape({
+      actionButtonLabel: React.PropTypes.string.isRequired,
+      message: React.PropTypes.string.isRequired,
+      state: React.PropTypes.string.isRequired,
```

Comment:
Kinda splitting hairs here, but I have to say I find `data.state` kind of akward. I  can't help but think of `state` as a *special* keyword in React. Can't we rename this to something else... "context", "severity", "level", "type"?